katello.dot

```diff
@@ -2,108 +2,120 @@
 
 [Browser] as browser
 
-node "server" {
-  [Postgres] as postgres
-  [Candlepin] as candlepin
+node "Main Environment" {
```

Comment:
What do you envision "Main Environment" meaning? This implies other environments exist, what are those others you forsee?

Reply:
Here I'm thinking about "Main Environment" == Satellite Server and "External Environment" == Capsules.